src/Workspaces/MSBuildTest/Resources/NetCoreApp2/Project.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Should we change the folder name to match? or maybe remove the version from the folder?